server/superdesk/data_consistency/compare_repositories_tests.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+import superdesk
+from eve.utils import ParsedRequest
+from superdesk.tests import TestCase
+from superdesk import get_resource_service
+from superdesk.data_consistency.compare_repositories import CompareRepositories
```

Comment:
Merging this PR is fine, yes.

Re: helper method/class - should be straightforward, we just add it to the `superdesk.tests.TestCase` and invoke it with the name of the class under test (as a string) that we want to import in a test module. Also thanks for taking care of opening a ticket.